starter-code/src/clue.js

```diff
@@ -1,94 +1,160 @@
-occupation:   Actor
+// missScarlet
+// first_name:   Kasandra
+// last_name:    Scarlet
+// color:        red
+// description:  She is an A-list movie star with a dark past
+// age:          31
+// image:        https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg
+// occupation:   Actor
 
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
+// mrsPeacock
+// first_name:   Eleanor
+// last_name:    Peacock
+// color:        blue
+// description:  She is from a wealthy family and uses her status and money to earn popularity
+// age:          36
+// image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
+// occupation:   Socialité
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+// mrMustard
+// first_name:   Jack
+// last_name:    Mustard
+// color:        yellow
+// description:  He is a former football player who tries to get by on his former glory
+// age:          62
+// image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
+// occupation:   Retired Football player
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+// name: rope        ---   weight: 10
+// name: knife       ---   weight: 8
+// name: candlestick ---   weight: 2
+// name: dumbbell    ---   weight: 30
+// name: poison      ---   weight: 2
+// name: axe         ---   weight: 15
+// name: bat         ---   weight: 13
+// name: trophy      ---   weight: 25
+// name: pistol      ---   weight: 20
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+// name: Dinning Room
+// name: Conservatory
+// name: Kitchen
+// name: Study
+// name: Library
+// name: Billiard Room
+// name: Lounge
+// name: Ballroom
+// name: Hall
+// name: Spa
+// name: Living Room
+// name: Observatory
+// name: Theater
+// name: Guest House
+// name: Patio
 
-// Characters Collection
-var charactersArray = [];
 
-// Rooms' Collection
-var roomsArray = [];
-
-// Weapons Collection
-var weaponsArray = [];
+//Characters
+var characterArray = [
```

Comment:
This is a bit "workaround" since we are practicing JS methods and this was good chance to create multiple objects and push them into the array (the same as in weaponsArray and roomsArray). This works too tho.